### ISSUE TITLE
Build failure with sbv-5.0

### ISSUE THREAD

**Issue by snoyberg** [GaloisInc/cryptol#275]
```
src/Cryptol/Symbolic/Value.hs:85:9:
    Not in scope: ‘svReduceInPathCondition’
```

**Comment by DanBurton** [comment]
Bump. The latest sbv is now 5.5, but cryptol uses < 5.2.

**Comment by TomMD** [comment]
There are several regressions I'd hope we fix before a release.  @acfoltzer Do you have a clear picture and/or should we talk at some point about point release and stoppers to the next major release?

**Comment by acfoltzer** [comment]
The main thing I want to take care of before cutting a 2.3 release from `master` is the constraint solver performance. If I have time I may backport our SBV updates and do another 2.2 release.

**Comment by LeventErkok** [comment]
@acfoltzer I think the master is now on >= 5.5; so this ticket should be good to close?

**Comment by acfoltzer** [comment]
It is, but we have not done a Hackage release for quite some time due to the performance of the new constraint solver.

**Comment by mietek** [comment]
This affects the 2.2.4 release.

**Comment by acfoltzer** [comment]
Fixed in [2.2.6](https://github.com/GaloisInc/cryptol/releases/tag/v2.2.6), thanks for your patience
